glances/plugins/containers/engines/docker.py

```diff
@@ -215,25 +215,26 @@ class DockerContainersExtension:
 
     CONTAINER_ACTIVE_STATUS = ['running', 'paused']
 
-    def __init__(self):
+    def __init__(self, server_urls:str|list = 'unix:///var/run/docker.sock'):
         if import_docker_error_tag:
             raise Exception("Missing libs required to run Docker Extension (Containers) ")
 
-        self.client = None
+        self.clients = None
         self.ext_name = "containers (Docker)"
         self.stats_fetchers = {}
 
-        self.connect()
+        self.connect(server_urls)
 
-    def connect(self):
+    def connect(self, base_urls:str|list = 'unix:///var/run/docker.sock'):
         """Connect to the Docker server."""
         # Init the Docker API Client
+        base_urls = [base_urls] if isinstance(base_urls, str) else base_urls
         try:
             # Do not use the timeout option (see issue #1878)
-            self.client = docker.from_env()
```

Comment:
test if base_urls is empty then use from_env() else use the base_urls list instead